config/deploy/production.rb

```diff
@@ -12,6 +12,7 @@
 role :db,  "#{fetch(:user)}@#{fetch(:server)}"
 
 set :repo_url, 'git@bitbucket.org:andreslemik/omanager.git'
+set :branch, 'new_interface'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.